endpoints/remotehost/remotehost

```diff
@@ -304,30 +381,40 @@ function launch_osruntime() {
 
     set_total_cpupart
 
+    # Launch the tool engines (support only pods)
+    local tools=`awk -F: '{print $1}' $config_dir/tool-cmds/profiler/start`
+    echo "Creating a pod for each of these tools: $tools"
+    tool_count=1
+    echo "Going to create a pod for each tool"
+    for tool in $tools; do
+        # This label will not work with mulitple remotehost endpoints!
```

Comment:
What do we have to do to fix this comment?  I'm actively using multiple remotehost endpoints right now so I guess this would break me?

Reply:
This is odd.  I thought I included a newer commit that fixes this. FWIW this is all draft

Reply:
And yes I do intend to support multiple remotehost endpoints to same host

Reply:
I think I just forgot to remove the comment which no longer applies

Reply:
BTW it looks like multiple osp or k8s endpoints are also broken, not from this code but just since they have existed.  For example, in the case of mulitple k8s, we would end up having a worker-1 engine from both endpoints.